### PR TITLE
Fixes #263: Add a toggle to make passwords visible in login activity

### LINKED ISSUE #263: Switch between hide and view password in LoginActivity
Currently there is no option to view the password whenever it is entered. There should be an option to view the password which is entered.

@timabbott @kunall17 Can i work on this? This is a very small issue. Will send the PR soon.

### PR DESCRIPTION
**Pull request by saketkumar95**
PR address [#263](https://github.com/zulip/zulip-android/issues/263)

Now there is toggle between **Show and Hide Password**

Demo video here : 

https://drive.google.com/file/d/0Bx-iZdT4RNS9ZF9iazJXdGNaY1k/view?usp=sharing

@timabbott  @kunall17 Review and let me know if there is any changes required. :)

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@saketkumar95, it looks like you've already signed the Dropbox CLA. Thanks!

**Comment by kunall17** [comment]
Showing a HIDE, SHOW label would start causing problems on a small screen phone or where passwords are large use an icon instead  https://material.io/icons/#ic_visibility

**Comment by saketkumar95** [comment]
@kunall17 Done! Any more changes?? :)

**Comment by kunall17** [comment]
@saketkumar95 remove the formatting changes in login.xml

**Comment by kunall17** [comment]
Also change the commit message!

**Comment by saketkumar95** [comment]
@kunall17 Done! :)

File: app/src/main/res/layout/login.xml [hunk 1]
```diff
@@ -52,75 +53,91 @@
...
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:text="@string/login_button" />
-
-        </LinearLayout>
+     <LinearLayout
+         android:id="@+id/serverFieldLayout"
+         android:layout_width="match_parent"
+         android:layout_height="wrap_content"
+         android:gravity="center"
```
Comment (kunall17) [inline app/src/main/res/layout/login.xml]: Too much format changes in here as well, fix this one too!

**Comment by kunall17** [comment]
Squash these two commits!

**Comment by saketkumar95** [comment]
@kunall17 Review now. I hope now nothing more is left now.

**Comment by kunall17** [comment]
@timabbott LGTM! :)

**Comment by timabbott** [comment]
Merged, thanks @saketkumar95 and @kunall17!!

**Pull request closed without merging**